ui/app/AppLayouts/Chat/ContactsColumn/EmptyView.qml

```diff
@@ -1,82 +1,177 @@
             }
-            SuggestedChannel {
-                channel: "status"
+            ColorOverlay {
+                anchors.fill: closeImg
+                source: closeImg
+                color: Style.current.darkGrey
             }
-            SuggestedChannel {
-                channel: "crypto"
+            MouseArea {
+                anchors.fill: closeImg
+                cursorShape: Qt.PointingHandCursor
+                onClicked: {
+                    emptyView.onCloseButtonPressed()
+                }
             }
-            SuggestedChannel {
-                channel: "tech"
+
+            StyledText {
+                id: chatAndTransactText
+                text: qsTr("Chat and transact privately with your friends")
+                anchors.top: parent.top
+                anchors.topMargin: 56
+                horizontalAlignment: Text.AlignHCenter
+                font.pixelSize: 15
+                wrapMode: Text.WordWrap
+                anchors.right: parent.right
+                anchors.rightMargin: Style.current.xlPadding
+                anchors.left: parent.left
+                anchors.leftMargin: Style.current.xlPadding
             }
-            SuggestedChannel {
-                channel: "music"
+
+            StyledButton {
+                label: qsTr("Invite friends")
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: Style.current.xlPadding
+                onClicked: {
+                    inviteFriendsPopup.open()
+                }
             }
-            SuggestedChannel {
-                channel: "movies"
+
+            InviteFriendsPopup {
+                id: inviteFriendsPopup
             }
-            SuggestedChannel {
-                channel: "test"
+        }
+
+        Separator {
+            anchors.topMargin: 0
+            anchors.top: inviteFriendsContainer.bottom
+        }
+
+        Item {
+            id: suggestionContainer
+            anchors.top: inviteFriendsContainer.bottom
+            anchors.right: parent.right
+            anchors.left: parent.left
+            height: suggestionsText.height + channelFlow.height + 2 * Style.current.xlPadding + Style.current.bigPadding
+
+            StyledText {
+                id: suggestionsText
+                width: parent.width
+                //% "Follow your interests in one of the many Public Chats."
+                text: qsTrId("follow-your-interests-in-one-of-the-many-public-chats.")
+                anchors.top: parent.top
+                anchors.topMargin: Style.current.xlPadding
+                font.pointSize: 15
+                wrapMode: Text.WordWrap
+                verticalAlignment: Text.AlignTop
+                horizontalAlignment: Text.AlignHCenter
+                fontSizeMode: Text.FixedSize
+                renderType: Text.QtRendering
+                onLinkActivated: console.log(link)
```

Comment:
?

Reply:
Lol, I was as confused as you, seems like it's not fully my fault, it was already like that on master

Reply:
removed